### ISSUE TITLE
🔴 Versionar o harness de avaliação por classe — a métrica que decide não é reproduzível

### ISSUE THREAD

**Issue by logikos33** [logikos33/Recognition#418]
## O problema

O baseline `mascara = 0,4375` do TREINO 1 (14/08) veio de um harness offline que **nunca foi commitado**. `per_class_eval_split` não existe em lugar nenhum do repositório:

```bash
grep -rn "per_class_eval_split" --include="*.py" .   # zero resultados
```

O `metrics.json` que o runner sobe para o R2 tem **157 bytes** — `framework`, `epochs`, `r2_key`. Nada por classe.

Resultado: a métrica que decide "promover ou não" existiu uma vez, num scratchpad de sessão, e evaporou.

## O que já existe para aproveitar

O harness foi **reconstruído e calibrado** em 18/08: reproduz `tp=14` e `fn=92` exatos do baseline em `thr=0.55` com casamento guloso cego à classe, IoU 0,5. Só o `fp` diverge (13 contra 18).

Componentes: pré-processamento RF-DETR (RGB, 560×560, /255, normalização ImageNet), `sigmoid` sobre logits, `topk` sobre query×classe, boxes `cxcywh` normalizados → `xyxy`, casamento guloso, matriz de confusão.

## Definição de pronto

- harness em `training/eval/` com CLI (`--model`, `--coco`, `--thr`)
- teste que fixa a calibração contra um par (modelo, gabarito) conhecido — instrumento novo se calibra contra valor conhecido **antes** da medida que decide
- o runner passa a chamá-lo e a subir o `metrics.json` completo

**Comment by logikos33** [comment]
Fechada pelo PR #430 (mergeado na `develop`, `bf5ac55b`).

Harness em `training/eval/per_class_eval.py`, calibrado contra o baseline conhecido antes de ser usado para decidir: reproduz `tp=14` e `fn=92` exatos do TREINO 1 em `thr=0.55`. A divergência que sobra (`fp` 13 contra 18) está documentada no cabeçalho do módulo, não escondida.

7 testes fixam a regra de casamento — que é a parte que decide o número.

*(Não fechou sozinha: o `Closes` só dispara em merge para a branch padrão, e o alvo aqui é `develop`.)*
